0.56.11 fails to install correctly

Hi, I cannot get 0.56.11 to install.

The following command
`npm install --save semantic-ui-react@0.56.11`
runs successfully, however, semantic-ui-react does not seem to actually be built thereafter. (There is no dist directory in ./node_modules/semantic-ui-react)

### Steps
1. Install version 0.56.11

`npm install --save semantic-ui-react@0.56.11`

### Expected Result
semantic-ui-react to have been importable.

### Actual Result
Cannot import from semantic-ui-react.
```                                                                                                                                                                                              
> require('semantic-ui-react')
Error: Cannot find module 'semantic-ui-react'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at repl:1:1
    at sigintHandlersWrap (vm.js:22:35)
    at sigintHandlersWrap (vm.js:96:12)
    at ContextifyScript.Script.runInThisContext (vm.js:21:12)
    at REPLServer.defaultEval (repl.js:313:29)
    at bound (domain.js:280:14)
> 
```

### Version
0.56.11

--- a/circle.yml
+++ b/circle.yml
@@ -20,6 +20,6 @@ deployment:
     commands:
       - git config --global user.email "devteam+deweybot@technologyadvice.com"
       - git config --global user.name "deweybot"
-      - $(npm bin)/ta-script npm/auto-release
+      - npm run auto-release
       - $(npm bin)/ta-script circle_ci/create_changelog
       - NODE_ENV=production npm run deploy:docs

--- a/package.json
+++ b/package.json
@@ -9,6 +9,8 @@
   ],
   "scripts": {
     "docs": "gulp docs",
+    "preauto-release": "npm run build",
+    "auto-release": "ta-script npm/auto-release",
     "build": "npm run build:commonjs && npm run build:umd && npm run build:docs",
     "prebuild:commonjs": "rimraf dist/commonjs && npm run build:dll",
     "build:commonjs": "babel src -d dist/commonjs",

